lib/rspec/rails/view_rendering.rb

```diff
@@ -85,22 +82,27 @@ def append_view_path(new_path)
       private
 
         def _path_decorator(path)
-          EmptyTemplatePathSetDecorator.new(ActionView::PathSet.new(Array.wrap(path)))
+          EmptyTemplateResolver.new(path)
         end
       end
 
       included do
         before do
           unless render_views?
-            @_empty_view_path_set_delegator = EmptyTemplatePathSetDecorator.new(controller.class.view_paths)
-            controller.class.view_paths = ::ActionView::PathSet.new.push(@_empty_view_path_set_delegator)
+            @_original_path_set = controller.class.view_paths
+
+            empty_template_path_set = @_original_path_set.map do |resolver|
+              EmptyTemplateResolver.new(resolver.to_s)
+            end
+
+            controller.class.view_paths = empty_template_path_set
             controller.extend(EmptyTemplates)
           end
         end
 
         after do
           unless render_views?
-            controller.class.view_paths = @_empty_view_path_set_delegator.original_path_set
+            controller.class.view_paths = @_original_path_set
```

Comment:
Hi! We've actually merged a travis passing-fix in 3.4.1, this fix is a forward-port of that. And I'll fix that tomorrow.